apps/frontend/src/modules/workspace/components/settings/Workspace/Member.tsx

```diff
@@ -43,23 +43,23 @@ export const Member = () => {
                 </div>
 
                 <div className="flex w-full items-end gap-2">
-                    <div className="w-[75%]">
+                    <div className="flex-1">
                         <TextInput
                             placeholder=""
                             value={addEllipsis(inviteLink, 60)}
                             disabled={true}
                             rightIcon={
                                 <button disabled={loading} onClick={handleInviteLinkRefresh}>
-                                    <RefreshCcwIcon size={20} className={loading ? "spinner__circle" : ""} />
+                                    <RefreshCcwIcon size={16} className={loading ? "spinner__circle" : ""} />
                                 </button>
                             }
                         />
                     </div>
-                    <div className="w-[25%]">
+                    <div>
                         <Button
                             text="Copy"
-                            className="text-sm"
-                            icon={<CopyIcon size={16} />}
+                            className=" h-[50px] w-[92px] p-3 text-sm"
```

Comment:
@afolasope We should not be modifying button styles directly. It makes no sense to have a design system then we start sprinkling custom styles across the codebase, it makes it very difficult to maintain. What we should rather have is the concept of `variant` like `sm`, `lg` `xl` e.t.c.